help.go

```diff
@@ -1,7 +1,9 @@
 package task
 
 import (
+	"encoding/json"
 	"fmt"
+	"github.com/go-task/task/v3/internal/editors"
```

Comment:
This should be moved to the second import group, together with `"github.com/go-task/task/v3/internal/logger"`.

Reply:
fixed in a rebased commit